packages/ui/src/components/ReviewArea.tsx

```diff
@@ -219,6 +220,34 @@ export default React.memo(function ReviewArea({
       ? currentItem?.colorPalette ?? items[items.length - 1].colorPalette
       : colors.palettes.red;
 
+  const onMemoizedMark = useCallback(
+    (outcome: PromptRepetitionOutcome) => {
+      if (!currentItem) {
+        throw new Error("Marking without a topmost item");
+      }
+      onMark({ reviewAreaItem: currentItem, outcome });
+    },
+    [currentItem, onMark],
+  );
+
+  // default action (Space)
+  useKey(" ", () => {
+    if (!isShowingAnswer) {
+      setShowingAnswer(true);
+    } else {
+      onMemoizedMark(PromptRepetitionOutcome.Remembered);
```

Comment:
Would you mind making this reference `actions[1]` so that the two actions don't accidentally drift apart?